pkg/kubectl/stop.go

```diff
@@ -203,7 +203,7 @@ func (reaper *ReplicationControllerReaper) Stop(namespace, name string, timeout
 	}
 	if len(overlapRCs) > 0 {
 		return fmt.Errorf(
-			"Detected overlapping controllers for rc %v: %v, please manage deletion individually with --cascade=false.",
+			"overlapped controllers are detected for rc %v: %v, please manage deletion individually with --cascade=false.",
```

Comment:
this command has been depreacted https://github.com/kubernetes/kubernetes/issues/11384, I will remove this file eventually in this pr https://github.com/kubernetes/kubernetes/pull/46927